bindata/network/ovn-kubernetes/ovnkube-node.yaml

```diff
@@ -267,6 +274,7 @@ spec:
             --loglevel "${OVN_KUBE_LOG_LEVEL}" \
             --inactivity-probe="${OVN_CONTROLLER_INACTIVITY_PROBE}" \
             ${gateway_mode_flags} \
+            --ovnkube-node-mode full \
```

Comment:
Shouldn't the default for this argument be `full`? Otherwise everyone has to update their daemonsets.

Reply:
Yes, the default ovnkube-node-mode is full if not specified. Do you prefer to remove it? I was thinking it may cause trouble if the default mode will be changed, but that is unlikely to happen as far as I can see.

Reply:
I will remove the `--ovnkube-node-mode full` from the `ovnkube-node.yaml`.